shared/AppInsightsCore/src/JavaScriptSDK.Enums/LoggingEnums.ts

```diff
@@ -92,6 +92,7 @@ export const _InternalMessageId = {
     InvalidEvent: 70,
     FailedMonitorAjaxSetRequestHeader: 71,
     SendBrowserInfoOnUserInit: 72,
-    PluginException: 73
+    PluginException: 73,
+    SnippetScriptLoadFailure: 99
```

Comment:
why is this not 74? 😬

Reply:
I just wanted to move it to something that was obvious and easy to recall from an ICM :-)